songpal/notification.py

```diff
@@ -153,23 +159,11 @@ def __str__(self):
 
 
 @attr.s
-class ContentChange(ChangeNotification):
+class ContentChange(ChangeNotification, PlayInfo):
     """This gets sent as a notification when the source changes."""
 
     make = classmethod(make)
 
-    contentKind = attr.ib()
-    service = attr.ib()
-    source = attr.ib()
-    output = attr.ib()
-    uri = attr.ib()
-    applicationName = attr.ib()
```

Comment:
Not all of these seem to be available in `PlayInfo`, at least `applicationName` seems to be missing.

Reply:
Turns out `applicationName` was the only one missing and it was meant to exist in `PlayInfo` as well.

Added as a pre-requisite commit.